src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/HttpApi/ApiClient.cs

```diff
@@ -320,10 +320,37 @@ public async Task<string> GetMetricsAsync(CancellationToken token)
             throw await CreateUnexpectedStatusCodeExceptionAsync(response).ConfigureAwait(false);
         }
 
+        public async Task<OperationResponse> EgressTraceAsync(int processId, int durationSeconds, string egressProvider, CancellationToken token)
+        {
+            string uri = FormattableString.Invariant($"/trace/{processId}?egressProvider={egressProvider}&durationSeconds={durationSeconds}");
```

Comment:
Just another case of using the old format -> "/trace?pid={processId}&..."

Reply:
Yep, I didn't fix these up after merging.